src/libraries/Microsoft.PowerFx.Json/FormulaValueJSON.cs

```diff
@@ -115,6 +108,11 @@ public static FormulaValue FromJson(JsonElement element, FormulaValueJsonSeriali
                         DateTime dt3 = element.GetDateTime();
                         return DateTimeValue.New(TimeZoneInfo.ConvertTimeToUtc(dt3));
                     }
+                    else if (formulaType is BlobType)
+                    {
+                        BlobElementBase resElem = new Base64Blob(element.GetString());
+                        return BlobValue.NewBlob(resElem);
+                    }                    

```

Comment:
This is asserting that if JSON expects a blob, we assume the element is a base64 encoded string?

Can we be sure that's always true?

Reply:
I'm also not sure we can make this assumption.